torch_ipex/csrc/init_python_bindings.cpp

```diff
@@ -110,38 +108,6 @@ void InitIpexModuleBindings(py::module m) {
           return AtenIpexTypeExt::embedding_bag_backward(grad, indices, offsets, offset2bag, bag_size, maximum_indices, num_weights, scale_grad_by_freq, mode, sparse, per_sample_weights);
         });
 
-  m.def("linear",
-        [](const at::Tensor& input, const at::Tensor& weight, const at::Tensor& bias) {
-          return AtenIpexTypeExt::linear(input, weight, bias);
-        });
-  m.def("linear_fuse_relu",
-        [](const at::Tensor& input, const at::Tensor& weight, const c10::optional<at::Tensor>& bias) {
-          return AtenIpexTypeExt::linear_fuse_relu(input, weight, bias);
-        });
-  m.def("linear_backward",
-        [](const at::Tensor& input, const at::Tensor& grad_output, const at::Tensor& weight, std::array<bool,3> output_mask) {
-          return AtenIpexTypeExt::linear_backward(input, grad_output, weight, output_mask);
-        });
-  m.def("relu_use_dst_backward",
-        [](const at::Tensor& grad_output, const at::Tensor& output) {
-          return AtenIpexTypeExt::relu_use_dst_for_bwd(grad_output, output);
-        });
-  m.def("adaptive_avg_pool2d",
-        [](at::Tensor const& input, at::IntArrayRef output_size) {
-          return AtenIpexTypeExt::adaptive_avg_pool2d(input, output_size);
-        });
-  m.def("adaptive_avg_pool2d_backward",
-        [](const at::Tensor& grad_output, const at::Tensor& input) {
-          return AtenIpexTypeExt::adaptive_avg_pool2d_backward(grad_output, input);
-        });
-  m.def("max_pooling",
-        [](const at::Tensor& input, at::IntArrayRef kernel_size, at::IntArrayRef stride, at::IntArrayRef padding, at::IntArrayRef dilation, bool ceil_mode) {
-          return AtenIpexTypeExt::max_pooling(input, kernel_size, stride, padding, dilation, ceil_mode);
-        });
-  m.def("max_pooling_backward",
-        [](const at::Tensor& grad_output, const at::Tensor& output, const at::Tensor& input, at::IntArrayRef kernel_size, at::IntArrayRef stride, at::IntArrayRef padding, at::IntArrayRef dilation, bool ceil_mode) {
-          return AtenIpexTypeExt::max_pooling_backward(grad_output, output, input, kernel_size, stride, padding, dilation, ceil_mode);
-        });
   m.def("reshape",
```

Comment:
Minor, lambda here could be simply replaced with a simple form
```
m.def("reshape", &AtenIpexTypeExt::reshape);
```
because it didn't do anything with the parameters, just do the forwarding. 

Same for `packed_add_`, `interaction_forward`, `interaction_backward`, `embedding_bag_forward`, `embedding_bag_backward` above.

I would appreciate if you could remove these code along with this PR. Thanks